Bricks/cerealize/cerealize.h

```diff
@@ -212,7 +204,7 @@ class CerealBinaryFileParser {
     if (fi_.peek() != std::char_traits<char>::eof()) {  // This is safe with any next byte in file.
       std::unique_ptr<T_ENTRY> entry;
       si_(entry);
-      processor(*entry.get());
+      processor(std::move(*entry.get()));
```

Comment:
Yes, it doesn't look perfect to me either. But it's legal, valid C++, and I don't know of a way that does the same while ensuring no memory will be leaked in any case.